### ISSUE TITLE
How many steps are needed in stage 1?

### ISSUE THREAD

**Issue by ireneMsm2020** [guoqincode/AnimateAnyone-unofficial#40]
I trained in stage_1, but the loss does not decrease?，is is correct?

**Comment by guoqincode** [comment]
batch size=64, steps=30000-60000, resolution=768*768

**Comment by ireneMsm2020** [comment]
batch size=64, is it means every A100 GPU set 8, and training is 8 A100?  I trained in 80G A100 GPU, the max batch_size can set in a single A100 is 10.

**Comment by guoqincode** [comment]
You can use accelerate or gradient accumulation, or 16 cards.
